source/jsonnet/northern-ireland/individual/blocks/personal-details/sex.jsonnet

```diff
@@ -52,12 +52,24 @@ local proxyTitle = {
         when: [rules.over16],
       },
     },
+    {
+      goto: {
+        block: 'marriage-type',
+        when: [rules.lastBirthdayAgeOver(16)],
```

Comment:
The spec is really confusing
```sex - if under 4 route to country-of-birth if 4 to 15 route to in-education (under 16 variant) if 16+ route to marriage-type```
Is 16+ including 16 and if so shouldn't the rule be lastBirthdayAgeOver(15) as read literally its over and not equals and over?

Reply:
In the card it is exclusive, so 4 to 15 is inclusive, 16+ is greater than or equal to 16 etc.

I brought up this point a while back about calling it `over` when it is actually inclusive, but it was agreed at the time that for age it would be fine to use `over` when it was also inclusive. 

Personally, when I think about just numbers, yes over sounds exclusive but for age I think `over` makes sense to be inclusive, as normally when you say `Are you over 18`, it is implied they are at least 18. So when talking about age if I have turned 18 then I am technically over 18.

This is what we already do with the existing DOB check, if we think it is not clear, we should propose something new and refactor it as a separate piece of work.

Reply:
It is confusing, as even though we are dealing with ages the type is still integer and I'm thinking of it in terms of dealing with integers, therefore over is to me exclusive of the lower bound. It should be a separate card to review and update everywhere in the schema that this occurs

Reply:
I agree, but happy for it to go into another story. I think its says it all when 2 reviews and the developer all have questioned the inclusiveness of over.